src/app.jsx

```diff
@@ -212,7 +212,7 @@ class AppActive extends React.Component {
         };
         const createVmAction = <CreateVmAction {...properties} mode='create' />;
         const importDiskAction = <CreateVmAction {...properties} mode='import' />;
-        const vmActions = <> {createVmAction} {importDiskAction} </>;
+        const vmActions = <> {importDiskAction} {createVmAction} </>;
```

Comment:
FWIW, I have no strong opinion about this (i.e. no veto), but it feels strange to me. It breaks muscle memory, both are secondary, and one is not obviously "more important" than the other.

Reply:
Default actions are more on the right.

You can create a VM without anything but Cockpit and Cockpit Machines. But you cannot import a VM unless you have a disk image somewhere, needing more from the user. Hence Create being on the right, as originally designed, and import being on the left. The fact that they were swapped in implementation is a bug (and a longstanding one).